Transformer/NgAdminWithRelationshipsTransformer.php

```diff
@@ -92,7 +93,7 @@ private function transformReferenceRelationships($configuration)
                 'name' => $field['name'],
                 'type' => 'reference',
                 'referencedEntity' => [
-                    'name' => $matchingAssociation['fieldName'],
+                    'name' => Inflector::pluralize($matchingAssociation['fieldName']),
```

Comment:
Long term, it might make more sense to look the name up from the registry using the class name.